src/gui/optionsdlg.cpp

```diff
@@ -648,7 +649,8 @@ void OptionsDialog::saveOptions()
     pref->setWebUiEnabled(isWebUiEnabled());
     if (isWebUiEnabled()) {
         pref->setServerDomains(m_ui->textServerDomains->text());
-        pref->setWebUiPort(webUiPort());
+        pref->setWebUiAddress(m_ui->textWebUiAddress->text());
```

Comment:
@fmatthew5876 
You are missing this:
`m_ui->textWebUiAddress->setText(pref->getWebUiAddress());`

Reply:
Fixed in latest rebase